references/sdk.md

```diff
@@ -303,25 +303,28 @@ Data is a dict object with the following schema:
 {
     "objective_key": str,
     "traintuples": list[{
+        "traintuple_id": str,
         "algo_key": str,
         "data_manager_key": str,
         "train_data_sample_keys": list[str],
-        "traintuple_id": str,
         "in_models_ids": list[str],
         "tag": str,
     }],
     "composite_traintuples": list[{
+        "composite_traintuple_id": str,
         "algo_key": str,
         "data_manager_key": str,
         "train_data_sample_keys": list[str],
         "in_head_model_id": str,
         "in_trunk_model_id": str,
         "out_trunk_model_permissions": {
+            "public": bool,
```

Comment:
The public will always be ignored, it will always be private. I think the documentation shouldn't mention a non existing parameter.

Reply:
I agree with you, it should not be there if it's ignored. But, I don't think that the public field is ignored. I looked at the chaincode implementation and it is handled like other permissions objects.

Reply:
True, but the backend always passes public set to False (`compositetraintuple.py` L45). So the user input is effectively ignored.

Reply:
Ok thanks, I will add a message then to explain this particular case.